server/core/config/raw/project.go

```diff
@@ -68,6 +86,13 @@ func (p Project) ToValid() valid.Project {
 	cleanedDir := filepath.Clean("./" + *p.Dir)
 	v.Dir = cleanedDir
 
+	if p.Branch != nil {
+		branch := *p.Branch
+		withoutSlashes := branch[1 : len(branch)-1]
```

Comment:
Why does the branch have to be surrounded by slashes?

Reply:
- To keep it consistent with server-side config that already has `branch` and [expects slashes around it if it's a regex](https://github.com/runatlantis/atlantis/blob/v0.20.1/server/core/config/raw/global_cfg.go#L203-L209)).
- To indicate to the user that a regular expression is expected here.

Reply:
Makes sense to me! Thank you.